services/database-analyzer.js

```diff
@@ -150,6 +151,10 @@ function DatabaseAnalyzer(databaseConnection, config, allowWarning) {
         'Your database is empty.',
         'Please, create some tables before running generate command.',
       );
+      await eventSender.notifyError('database_empty', 'Your database is empty.', {
+        orm: 'sequelize',
```

Comment:
[Misc] Why do we need to send this ?

Reply:
For event tracking/to have more precise error ;)